core_v2/sources/domains.move

```diff
@@ -578,24 +576,24 @@ module aptos_names_v2::domains {
     public entry fun renew_domain(
```

Comment:
how about rename function name to `set_domain_expiration` or `update_domain_expiration`?

Reply:
we still `renew` domains within the renewable window, but we `set` subdomain expirations to give domain owner more flexibility.